src/common/components/Tooltip/Skin/index.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import React from 'react';
+import Icon from 'common/components/Icon';
+import Background from '../Background';
+import styles from './styles.less';
+
+type Props = {
+  data: Object,
+};
+
+const SkinTooltip = ({ data: { skin } }: Props) => {
+  return (
+    <Background>
+      <div className={styles.itemHeader}>
```

Comment:
one alternative we could have done here is reuse `<ItemHeader />` and just give it a rarity of `white`, which would make it have white text :-)..

Reply:
That was actually my first approach, but it didnt work. I will put some more time into it.